### ISSUE TITLE
Value Range for Lab Spike % Recovery

### ISSUE THREAD

**Issue by ben-wetherill** [massbays-tech/MassWateR#60]
This is a minor and rare situation, and there are ways to work around it, but the package is not currently handling the value range for lab spikes correctly.  Would this be easy to handle?

For the result unit of measure, we have one special exception:  Lab Spikes may be entered with a percent unit of measure "%" or "% recovery" instead of the base unit of measure.  If the value ranges are defined so that values around 100 have a different DQO, then the wrong DQO might be used by the package.  In the attached files, both lab spikes for Nitrite should be misses, but only one is flagged.  I think the logic should be that if lab spikes are entered with the "%" or "% recovery" unit of measure, then the lab spike should be evaluated against the upper value range, and if the upper value range is not a % DQO, then tabMWRacc() should return an error.

[Sample_DQOAccuracy_testing.xlsx](https://github.com/massbays-tech/MassWateR/files/12551284/Sample_DQOAccuracy_testing.xlsx)
[Sample_Results_testspikes.xlsx](https://github.com/massbays-tech/MassWateR/files/12551285/Sample_Results_testspikes.xlsx)
[Sample_DQOFreqComp_testingspikes.xlsx](https://github.com/massbays-tech/MassWateR/files/12551286/Sample_DQOFreqComp_testingspikes.xlsx)

**Comment by ben-wetherill** [comment]
I'm very surprised to hear that % recovery was not evaluated as an absolute value because that is a fundamental piece of the % recovery logic.  I'm certain I would have tested that, but you are right that the current production version of MassWateR does not flag negative % recovery exceedances.  Is it possible that this changed somewhere along the way?  

The fixes look good mostly, but **I did find one issue**.  It is also related to the absolute value concept:
In the files I provided, if you change the Nitrite lab spike result value from 460 to 340, tabMWRacc() does not flag it as a miss.  It should because the absolute value of the difference is 60, which is greater than 50.

So, it turns out that spike/check accuracy was not evaluating absolute value for anything (% or normal units of measure).  I am 100% certain that I tested this before, so something must have changed.
